logicalRequirements.md

```diff
@@ -79,43 +72,206 @@ An `enemyKill` object can have the following properties:
 
 __Example:__
 ```json
-"requires":[
-  {"enemyKill":{
-    "enemies": [
-      [
-        "Yellow Space Pirate (wall)",
-        "Yellow Space Pirate (standing)"
-      ],
-      [
-        "Yellow Space Pirate (wall)"
-      ]
+{"enemyKill":{
+  "enemies": [
+    [
+      "Yellow Space Pirate (wall)",
+      "Yellow Space Pirate (standing)"
     ],
-    "excludedWeapons": ["Bombs"],
-  }}
-]
+    [
+      "Yellow Space Pirate (wall)"
+    ]
+  ],
+  "excludedWeapons": ["Bombs"]
+}}
 ```
 Since Yellow Space pirates have 900 health and are immune to uncharged beam shots, this object would be fulfilled by either Charge, Screw Attack, 27 Missiles, 9 Supers, or Morph + 6 Power Bombs (3 per group, expecting that they will double-hit for 400 damage each).
 
-### obstacle object
+__Additional considerations:__
+
+When trying to determine the ammo needed to fulfill an `enemyKill` object, it could be a good idea to factor in a player accuracy variable, to represent different levels of leniency.
+
+### Health Management Objects
+This section contains logical elements that are fulfilled by spending health. Encountering enough of those successively without having a chance to refill can impact the number of e-tanks that are needed to get through.
+
+#### acidFrames object
+An `acidFrames` object represents the need for Samus to spend time (measured in frames) in a pool of acid. This is meant to be converted to a flat health value based on item loadout. The vanilla damage for acid is 6 damage every 4 frames, halved by Varia (3 damage every 4 frames), and halved again by Gravity Suit (3 damage every 8 frames).
+
+__Example:__
+```json
+{"acidFrames": 35}
+```
+
+#### enemyDamage object
+An `enemyDamage` object represents the need for Samus to intentionally take damage from an enemy. This is meant to be converted to a flat health value based on item loadout. It has the following properties:
+* _enemy:_ The name of the enemy that will damage Samus
+* _type:_ The name of the attack that Samus will take damage from
+* _hits:_ The number of hits Samus will take from that attack
+
+__Example:__
+```json
+{"enemyDamage": {
+  "enemy": "Sidehopper",
+  "type": "contact",
+  "hits": 3
+}}
+```
+
+#### heatFrames object
+A `heatFrames` object represents the need for Samus to spend time (measured in frames) in a heated room. This is meant to be converted to a flat health value based on item loadout. The vanilla damage for heated rooms is 1 damage every 4 frames, negated by Varia or Gravity Suit.
+
+__Example:__
+```json
+{"heatFrames": 100}
+```
+
+#### hibashiHits object
```

Comment:
Nice picking this up. Now the metroidFrames ^^

Reply:
That probably shouldn't be a logical element. It's an attack performed by Metroids. The randomizer suits reduce it like enemy damage, too.

Reply:
Draygon electricity frames are less clear. I'll take a finer look and possibly add them in the doc.